packages/serverpod/lib/src/database/database_connection.dart

```diff
@@ -74,6 +78,7 @@ class DatabaseConnection {
       timeoutInSeconds: 60,
       transaction: transaction,
       include: include,
+      viewTable: viewTable,
```

Comment:
Question: Shouldn't the query be replaced here?

Reply:
> Question: Shouldn't the query be replaced here?

Sorry, I need some clarification, do you mean querying on line 72 should replace?